Vagrantfile

```diff
@@ -27,6 +27,8 @@ sudo service docker restart
 echo 'cd ~/go/src/github.com/cilium/cilium' >> /home/vagrant/.bashrc
 sudo -E /usr/local/go/bin/go get github.com/cilium/go-bindata/...
 sudo -E /usr/local/go/bin/go get -u github.com/google/gops
+sudo -E /usr/local/go/bin/go get -d github.com/lyft/protoc-gen-validate
+(cd ~/go/src/github.com/lyft/protoc-gen-validate ; make build)
```

Comment:
Should we bind it to a specific version or you want always the latest?

Reply:
Good call. That project doesn't do any releases or branches. I'll just pick the current master head, which is working for us.